src/Neo.CLI/CLI/MainService.Plugins.cs

```diff
@@ -90,6 +90,26 @@ private static async Task<Stream> DownloadPluginAsync(string pluginName, Version
                     s["tag_name"]!.GetValue<string>() == $"v{pluginVersion.ToString(3)}" &&
                     s["prerelease"]!.GetValue<bool>() == prerelease) ?? throw new Exception($"Could not find Release {pluginVersion}");
 
+            if (jsonRelease == null)
+            {
+                // If the corresponding version of the plugin is not found, get the latest version
+                jsonRelease = json.AsArray()
+                    .OrderByDescending(s => Version.Parse(s["tag_name"]!.GetValue<string>()?.TrimStart('v')))
+                    .FirstOrDefault();
+
+                if (jsonRelease != null)
+                {
+                    var latestVersion = Version.Parse(jsonRelease["tag_name"]!.GetValue<string>().TrimStart('v'));
+                    if (latestVersion < pluginVersion)
+                    {
+                        // If the latest version is lower than the locally passed version, use https://github.com/neo-project/neo-modules/releases/latest/download to get the latest version
+                        var latestDownloadUrl = $"https://github.com/neo-project/neo-modules/releases/latest/download/{pluginName}.zip";
```

Comment:
We need to change the url to neo now, also for `DownloadUrl`, could you do it @Jim8y ?